### DISCUSSION TITLE
Help with deployment

### DISCUSSION THREAD

**Discussion by xpatrykk** [vercel/next.js#51791] [Help]
### Summary

Hello,
I want to deploy a Next.js application (latest version) on an linux VPS with Nginx server.
I already have the proxy configuration figured out, and everything works locally.

I'm still wondering about the file copying aspect after building the application.

The question is, which files should be present on the server?
Is it sufficient to copy only these folders:

/node_modules
package.json
.next

or should I place the entire repository on the server, including the source code from the src folder?
My application uses SSR and ISR.

Thanks!

**Comment by jsebbari** [comment]
Hi, do you fix this problem?

Thank you
